docs/pkg.8

```diff
@@ -235,10 +235,10 @@ Fetch packages from a remote repository.
 .It Ic info
 Display information about installed packages and package files.
 .It Ic install
-Install a package from a remote package repository.
-If a package is found in more than one remote repository,
-then installation happens from the first one.
-Downloading a package is tried from each package repository in turn,
+Install a package from a package repository or a local archive.
```

Comment:
```suggestion
Install a package from a repository or a local archive.
```

In this context, does describing a repository as a package repository add value?